__tests__/basic.js

```diff
@@ -9,21 +11,19 @@ import webChatLoaded from './setup/conditions/webChatLoaded';
 // https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_WebDriver.html
 
 test('setup', async () => {
-  const { driver } = await setupWebDriver();
+  const { driver, pageObjects } = await setupWebDriver();
 
-  await driver.wait(webChatLoaded(), 2000);
-  await driver.wait(directLineConnected(), 2000);
+  await driver.wait(webChatLoaded(), timeouts.navigation);
+  await driver.wait(directLineConnected(), timeouts.directLine);
 
-  const input = await driver.findElement(By.tagName('input[type="text"]'));
+  const input = await driver.findElement(By.css('input[type="text"]'));
```

Comment:
question: Why do we want to use `css` instead of `tagName`?

Reply:
`tagName` should be just `input`, it does not support attribute `role`.

In contrast, `css` is the CSS selector, which could be `input[type="text"]`.